docs/api/index.rst

```diff
@@ -10,56 +10,56 @@ For those of you who want the gritty details.
 .. toctree::
     :maxdepth: 1
 
-    lexer
-    parser
     ast
     execer
     imphooks
+    lexer
+    parser
```

Comment:
I am not a huge of this, personally, because it doesn't reflect the logic of the system. Also, it doesn't relate to what the reader will see, since this is replaced by the title.